migration/src/main/scala/akka/persistence/postgres/migration/journal/JournalSchema.scala

```diff
@@ -110,12 +110,19 @@ private[journal] trait JournalSchema {
             DECLARE
             BEGIN
               INSERT INTO #$fullTableName (#$persistenceId, #$maxSequenceNumber, #$maxOrdering, #$minOrdering)
-              VALUES (NEW.#$jPersistenceId, NEW.#$sequenceNumber, NEW.#$ordering, NEW.#$ordering)
+              VALUES (
+                NEW.#$jPersistenceId,
+                NEW.#$sequenceNumber,
+                NEW.#$ordering,
+                CASE
+                  WHEN NEW.#$sequenceNumber = 1 THEN NEW.#$ordering
+                  ELSE -1
```

Comment:
@tiagomota I would use the `-1` market everywhere. It makes it explicit that it's not the actual `0` ordering. All the queries should work with -1 too right?

Reply:
Yup, I'm ok with that :)